app/components/DepartureTimeHr.tsx

```diff
@@ -0,0 +1,15 @@
+function DepartureTimeHr(props: { departure: any }) {
+
+    const {departure} = props
+
+    const {departureDateTime} = departure
+    const departureTime = new Date(departureDateTime).toLocaleTimeString('fr-EU', { hour: '2-digit', minute: '2-digit' })
```

Comment:
In this doc you will find everything thats avail as otpions for Intl formatting : https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat
(keep it somewhere to have ot close when needed)

In this case you can also just use `timeStyle: "short'` but what you did is also great !

Reply:
Thanks for the link. I save it in my favourite.